examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,102 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are stored in previous post processing pulled from an H5 file. The inverse of the T1(p) data is taken to produce the R1(p) data which is fit using the polyfit of the inverse of krho with two degrees of freedom. The R1(p) fit is then fed into the fitting routine for the enhancement data prior to normalization. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex
+from scipy.io import loadmat
+
+h5_file = "ras.h5"
+nodename = "230706_M67_a"
+# {{{ load data
+Ep = find_file(f"{h5_file}", exp_type="AG_processed_data", expno=f"{nodename}/Ep")
+R1p = find_file(f"{h5_file}", exp_type="AG_processed_data", expno=f"{nodename}/R1p")
+# }}}
+# {{{Find the index where the return powers begin
+flip_idx = np.where(np.diff(Ep.fromaxis("power").data) < 0)[0][0] + 1
+# }}}
+with figlist_var() as fl:
+    # {{{Plot Ep
+    fl.next("Integrated Enhancement")
+    fl.plot(
+        Ep["power", :flip_idx],
+        "o",
+        label="Progressive Saturation Data",
+        capsize=6,
+        alpha=0.5,
+    )
+    fl.plot(
+        Ep["power", flip_idx:],
+        "rx",
+        label="Returning Power Check",
+        capsize=6,
+        alpha=0.5,
+    )
+    # }}}
+    # {{{ plot R1p data
+    fl.next(r"$R_{1}(p)$")
+    fl.plot(R1p, "o", label="Experimental Data")
+    # }}}
+    # {{{load in T100 dataset
+    T10_p = loadmat(
+        search_filename(
+            "T10_DI_water_230412", exp_type="AG_processed_data", unique=True
+        )
+    )["a"][0, :]
+    R10_p = R1p.fromaxis("power").eval_poly(T10_p, "power") ** -1
+    # }}}
+    # {{{ fit krho inverse with two degrees of freedom and then apply to fit R1p
+    krho_inv = Ep.get_prop("acq_params")["concentration"] / (R1p - R10_p)
+    krho_inv_fit = krho_inv.polyfit("power", order=1)
+    krho_inv_fine = R1p.fromaxis("power").eval_poly(krho_inv_fit, "power")
+    R1p_fit = R10_p + Ep.get_prop("acq_params")["concentration"] / krho_inv_fine
+    fl.plot(R1p_fit, ls=":", color="k", label="Fit", alpha=0.5)
+    plt.ylabel(r"$R_{1} / s^{-1}$")
+    plt.xlabel("Power / W")
+    # }}}
+    # {{{ Fit E(p)
+    fl.next("Integrated Enhancement")
+    M0, A, phalf, p = symbols("M0 A phalf power", real=True)
+    sp = p / (p + phalf)
+    Ep_fit = lmfitdata(Ep["power", :flip_idx])
+    # Symbolic expression for Ep that is used in the symbolic function for the fitting of E(p)
+    Ep_fit.functional_form = M0 - ((M0 * A * sp) / (T10_p[0] + T10_p[1] * p) ** -1 + (
```

Comment:
There's a problem with your parentheses here, so that you don't end up doing the correct equation. You want to add together their relaxation rates, and divide your second term here by the relaxation rates (because it's supposed to be the term that involves the relaxivity times the T1)

Reply:
I would actually break this into two lines.... Construct the symbolic expression for the relaxation on the previous line, and then include it here.